File: backend/models/domain/Account.py
from flask_restful import fields


class Account():
    resource_fields = {
        'id': fields.Integer,
        'name': fields.String,
        'description': fields.String,
        'total': fields.Float,
        'color': fields.String,
    }

    def __init__(self, id=None, name=None, description=None, total=None, color=None):
        self.id = id
        self.name = name
        self.description = description
        self.total = total
        self.color = color

File: frontend/src/app/modules/accounts/account.ts
export class Account {

  public id: number;
  public name: string;
  public description: string;
  public total: number;
  public color: string;

  constructor(id: number, name: string, description: string, total: number, color: string) {
    this.id = id;
    this.name = name;
    this.description = description;
    this.total = total;
    this.color = color;
  }
}
